build.yaml

```diff
@@ -63,16 +86,25 @@ steps:
     - batch_pods_ns
  - kind: buildImage
    name: base_image
-   dockerFile: docker/Dockerfile.base
-   contextPath: .
+   dockerFile: /io/repo/docker/Dockerfile.base
+   contextPath: /io/repo
    publishAs: base
+   inputs:
+    - from: /repo
+      to: /io/repo
```

Comment:
So we shouldn't copy the whole thing.  The docker path is only the root because it needs pylintrc and setup.cfg.  We could put them in some etc directory or just copy the two files.

Reply:
Yeah, it's also way too slow to copy every file out or in, so I added https://github.com/hail-is/hail/pull/7626 to address this. Transfer as a single tar file, extract only the files you need.